src/templates/base.html

```diff
@@ -65,7 +65,12 @@
 								</svg>
 								Add Device
 							</a>
-						</li>						
+						</li>
+						<li class="nav-item">
+							<a class="nav-link active" aria-current="page" href="{% url 'account-settings' %}">
```

Comment:
please add an icon to the menu entry. This one should fit: https://icons.getbootstrap.com/icons/person-gear/